ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml

```diff
@@ -308,13 +308,11 @@ objects:
           clusterNetworks:
           - cidr:             10.128.0.0/14
             hostSubnetLength: 9
-          machineCIDR: 10.0.0.0/16
```

Comment:
Not sure why is this being removed

Reply:
> Not sure why is this being removed

Because I'm getting sloppy ;).  Fixed with 0e76dcf -> 3803c5e.